api/src/main/java/com/cloud/storage/ImageStore.java

```diff
@@ -21,6 +21,13 @@
 
 public interface ImageStore extends Identity, InternalIdentity {
 
+    String ACS_PROPERTY_PREFIX = "ACS-property-";
+    String REQUIRED_NETWORK_PREFIX = "ACS-network-";
+    String DISK_DEFINITION_PREFIX = "ACS-disk-";
+    String OVF_HARDWARE_CONFIGURATION_PREFIX = "ACS-configuration-";
+    String OVF_HARDWARE_ITEM_PREFIX = "ACS-hardware-item-";
+    String OVF_EULA_SECTION_PREFIX = "ACS-eula-";
+
```

Comment:
Minor nit - @nvazquez could this use a lowererd-case dot-joined like many other cloudstack such as `acs.property.`?

Reply:
The only pushback is that changing these keys will need a new Primate PR as the deployment wizard filters the details by these keys. If required I'll proceed with both